activemodel/lib/active_model/callbacks.rb

```diff
@@ -68,6 +68,10 @@ def self.extended(base) #:nodoc:
       end
     end
 
+    def self.verify_options(method, options) # :nodoc:
```

Comment:
The `method` argument is no longer used here.

This code is now simple enough that it could probably just be inlined.

Reply:
Agreed